index.js

```diff
@@ -38,8 +38,9 @@ got.stream(cfg.wolke)
 		// Compute average positions by time
 		const centers = normalized[line] = []
 		for (let i = 0; i < maxLength; i++) {
+
 			const position = center(trips.map((trip) => ({lat: trip[i].latitude, lon: trip[i].longitude}))
-			centers.push(Object.assign(position, {relative: i * cfg.interval})))
+			centers.push(Object.assign({latitude: position.lat, longitude: position.lon, relative: i * cfg.interval})))
```

Comment:
Now, `Object.assign` can be removed. I used it because `center()` may return more than those two keys `lat` and `lon`.